dev/ci/platforms/config.hera

```diff
@@ -8,11 +8,11 @@
 #########################################################################
 
 # Main CI root directory - Base directory for all CI/CD operations
-export GFS_CI_ROOT=/scratch1/NCEPDEV/global/glopara/GFS_CI_CD
+export GFS_CI_ROOT=/scratch3/NCEPDEV/global/role.glopara/GFS_CI_CD
```

Comment:
This update changes the path of JENKINS_AGENT_LAUNCH_DIR below on line 23.  We will need to physically move the directory where launching of the agent is maintained.  Let me do that first  before running the CI test on this PR.